### PR TITLE
Useful Resources subchapter for remote collab

### LINKED ISSUE #1666: Create a reference page for Remote Collaboration Chapter
### Summary

* We have a chapter on Remote Collaboration: https://the-turing-way.netlify.app/collaboration/remote-collab.html
* It needs a further reading section to add all the guides, papers and tutorials that have been released recently. In this issue, we want to capture those.

### What needs to be done?

- [ ] Test this and delete later

### Who can help?

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by harisood**
Fixes #1666

### CODE REVIEW

**Comment by harisood** [comment]
Did I do this right?

File: book/website/_toc.yml [hunk 1]
```diff
@@ -554,6 +554,8 @@ parts:
                 file: collaboration/remote-collab/remote-collab-guidelines
               - title: Managing Distributed Teams
                 file: collaboration/remote-collab/remote-collab-team
+              - title: Useful Resources
+                file: collaboration/remote-collab-useful-resources
```
Comment (harisood) [inline book/website/_toc.yml:558]:
```suggestion
                file: collaboration/remote-collab/useful-resources
```

Comment (aleesteele) [inline book/website/_toc.yml]:
```suggestion
                file: collaboration/remote-collab/remote-collab-useful-resources
```

**Review by aleesteele — APPROVED** [review]
Looks great @harisood ! Ready to merge 🚀

**Pull request merged**